entrypoint.py

```diff
@@ -205,6 +205,21 @@ def _localize(path, config_dir):
     return (config_dir / path).absolute().as_posix()
 
 
+def exists_python_code(dir):
+
+    py_files = ["setup.py", "tox.ini", "pyproject.toml", "requirements.txt", "requirements-dev.txt"]
+
+    for file in os.listdir(dir):
+        for py in py_files:
+            if file == py:
+                return True
+
+        if file.endswith(".py"):
+            return True
```

Comment:
Since `safety` analyzes python dependencies, we cannot just use it with single python files.